docs/quick_start.rst

```diff
@@ -0,0 +1,10 @@
+.. _quick_start:
+
+Quick Start
+===========
```

Comment:
Open to suggestions. For now, you can think of it as a temporary page for the template.